proto/proto/test_context.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
```

Comment:
Actual definition will come in followup. Just wanted to have the skeleton in place.